🐞 - `ng add taiga-ui` throws `[ERROR]: Could not find the project main file` + no `TuiRootModule` (Angular 17)

### Playground Link

See description

### Description

**Run console commands:**

<details><summary>ng version</summary>
<p>

```
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 17.0.0
Node: 18.15.0
Package Manager: npm 9.5.0
OS: darwin x64

Angular: 
... 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1700.0 (cli-only)
@angular-devkit/core         17.0.0 (cli-only)
@angular-devkit/schematics   17.0.0 (cli-only)
@schematics/angular          17.0.0 (cli-only)

```
</p>
</details> 

<details><summary>ng new ng-sandbox</summary>
<p>

```
? Which stylesheet format would you like to use? Less   [ http://lesscss.org                                             ]
? Do you want to enable Server-Side Rendering (SSR) and Static Site Generation (SSG/Prerendering)? No
CREATE ng-sandbox/README.md (1063 bytes)
CREATE ng-sandbox/.editorconfig (274 bytes)
CREATE ng-sandbox/.gitignore (548 bytes)
CREATE ng-sandbox/angular.json (2786 bytes)
CREATE ng-sandbox/package.json (1041 bytes)
CREATE ng-sandbox/tsconfig.json (909 bytes)
CREATE ng-sandbox/tsconfig.app.json (263 bytes)
CREATE ng-sandbox/tsconfig.spec.json (273 bytes)
CREATE ng-sandbox/.vscode/extensions.json (130 bytes)
CREATE ng-sandbox/.vscode/launch.json (470 bytes)
CREATE ng-sandbox/.vscode/tasks.json (938 bytes)
CREATE ng-sandbox/src/main.ts (250 bytes)
CREATE ng-sandbox/src/favicon.ico (15086 bytes)
CREATE ng-sandbox/src/index.html (295 bytes)
CREATE ng-sandbox/src/styles.less (80 bytes)
CREATE ng-sandbox/src/app/app.component.less (0 bytes)
CREATE ng-sandbox/src/app/app.component.html (20887 bytes)
CREATE ng-sandbox/src/app/app.component.spec.ts (928 bytes)
CREATE ng-sandbox/src/app/app.component.ts (372 bytes)
CREATE ng-sandbox/src/app/app.config.ts (227 bytes)
CREATE ng-sandbox/src/app/app.routes.ts (77 bytes)
CREATE ng-sandbox/src/assets/.gitkeep (0 bytes)
✔ Packages installed successfully.
    Successfully initialized git.
```

</p>
</details> 

<details><summary>cd ng-sandbox</summary></details> 

<details open><summary>ng add taiga-ui</summary>
<p>

```
ℹ Using package manager: npm
✔ Found compatible package version: taiga-ui@3.53.0.
✔ Package information loaded.

The package taiga-ui@3.53.0 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
? Do you want to use global Taiga UI classes, such as 'tui-space', 'tui-skeleton', etc? No
? Do you want to use Taiga UI dialogs? Yes
? Do you plan on using custom SVGs/icons or TUI editor? If `Yes` we will include ng-dompurify as sanitizer, read more: https://taiga-ui.dev/icons/bundled#sanitizer No
? Do you want to use Taiga UI alerts? Yes
? Choose additional packages to install 
    The main packages will be installed - core,cdk,kit,icons
UPDATE package.json (1172 bytes)
    [ERROR]: Could not find the project main file
    Content of the app was wrapped with tui-root component in /src/app/./app.component.html
UPDATE angular.json (3106 bytes)
UPDATE src/app/app.component.html (20910 bytes)
✔ Packages installed successfully.

```
</p>
</details> 

**it throws: `[ERROR]: Could not find the project main file`!**


Then try to run application:

<details open><summary>npm start</summary>
<p>

```

> ng-sandbox@0.0.0 start
> ng serve

⠼ Building...✘ [ERROR] NG8001: 'tui-root' is not a known element:
1. If 'tui-root' is an Angular component, then verify that it is included in the '@Component.imports' of this component.
2. If 'tui-root' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@Component.schemas' of this component to suppress this message. [plugin angular-compiler]

    src/app/app.component.html:0:0:
      0 │ 
        ╵ ^

  Error occurs in the template of component AppComponent.

    src/app/app.component.ts:9:15:
      9 │   templateUrl: './app.component.html',
        ╵                ~~~~~~~~~~~~~~~~~~~~~~


Application bundle generation failed. [3.692 seconds]
Watch mode enabled. Watching for file changes...

```

</p>
</details> 

**There is no `TuiRootModule` anywhere!**

### Angular version

17.0.0

### Taiga UI version

3.53.0

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

--- a/projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts
+++ b/projects/cdk/schematics/ng-add/steps/add-taiga-modules.ts
@@ -50,7 +50,7 @@ export function addTaigaModules(options: TuiSchema): Rule {
         }
 
         const buildOptions = getProjectTargetOptions(project, `build`);
-        const mainFilePath = buildOptions.main as string;
+        const mainFilePath = (buildOptions.main || buildOptions.browser) as string;
 
         if (!mainFilePath) {
             context.logger.error(`[ERROR]: Could not find the project main file`);

